include/ttmlir/Dialect/TT/IR/TTOpsEnums.td

```diff
@@ -72,12 +72,12 @@ def TT_MemorySpace : I32EnumAttr<"MemorySpace", "TT MemorySpace",
   let cppNamespace = "::mlir::tt";
 }
 
-def TT_TensorMemoryLayoutNone : I32EnumAttrCase<"None", 0, "none">;
-def TT_TensorMemoryLayoutInterleaved : I32EnumAttrCase<"Interleaved", 1, "interleaved">;
-def TT_TensorMemoryLayoutSingleBank : I32EnumAttrCase<"SingleBank", 2, "single_bank">;
-def TT_TensorMemoryLayoutHeightSharded : I32EnumAttrCase<"HeightSharded", 3, "height_sharded">;
-def TT_TensorMemoryLayoutWidthSharded : I32EnumAttrCase<"WidthSharded", 4, "width_sharded">;
-def TT_TensorMemoryLayoutBlockSharded : I32EnumAttrCase<"BlockSharded", 5, "block_sharded">;
+def TT_TensorMemoryLayoutInterleaved : I32EnumAttrCase<"Interleaved", 0, "interleaved">;
+def TT_TensorMemoryLayoutSingleBank : I32EnumAttrCase<"SingleBank", 1, "single_bank">;
+def TT_TensorMemoryLayoutHeightSharded : I32EnumAttrCase<"HeightSharded", 2, "height_sharded">;
+def TT_TensorMemoryLayoutWidthSharded : I32EnumAttrCase<"WidthSharded", 3, "width_sharded">;
+def TT_TensorMemoryLayoutBlockSharded : I32EnumAttrCase<"BlockSharded", 4, "block_sharded">;
+def TT_TensorMemoryLayoutNone : I32EnumAttrCase<"None", 5, "none">;
```

Comment:
See comment below regarding this.  I think we should always treat our enums separately and use switch-case statements to convert between them.